src/uwtools/api/file.py

```diff
@@ -0,0 +1,46 @@
+"""
+API access to uwtools file-management tools.
+"""
+from pathlib import Path
+from typing import List, Optional
+
+from uwtools.file import FileCopier as _FileCopier
+from uwtools.file import FileLinker as _FileLinker
+
+
+def copy(
+    target_dir: Path,
+    config_file: Optional[Path] = None,
+    keys: Optional[List[str]] = None,
+    dry_run: bool = False,
+) -> bool:
+    """
+    Copy files.
+
+    :param target_dir: Path to target directory
+    :param config_file: Path to YAML config file (defaults to stdin)
+    :param keys: YAML keys leading to file dst/src block
+    :param dry_run: Do not copy files
+    :return: True if no exception is raised
+    """
+    _FileCopier(target_dir=target_dir, config_file=config_file, keys=keys, dry_run=dry_run).go()
+    return True
```

Comment:
Do you think it might be worth adding a dict as an input option for the API? If so, I think it would make sense for the caller to pass the appropriate block. This could also be added as a feature later.

Reply:
Good idea. I'll either implement it in this PR or add a backlog task for it.

Reply:
Either way is fine by me.

Reply:
I implemented the `dict` option for configuring the file copy/link tool via the API in this PR.